roles/build/templates/deploy-k8s-community-user-db.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
The same use-case described above. Right now I placed nothing (5% dependency) of already depended role.